subprojects/internal-testing/src/main/groovy/org/gradle/test/fixtures/file/AbstractTestDirectoryProvider.java

```diff
@@ -99,16 +94,32 @@ void doCall() throws IOException {
                     });
                 }
             } catch (Exception e) {
-                String message = "Couldn't delete test dir for " + displayName + " (test is holding files open). "
-                    + "In order to find out which files are held open you may find http://file-leak-detector.kohsuke.org/ useful.";
-                if (suppressCleanupErrors) {
-                    System.err.println(message);
+                if (suppressCleanupErrors()) {
+                    System.err.println(cleanupErrorMessage());
                     e.printStackTrace(System.err);
                 } else {
-                    throw new GradleException(message, e);
+                    throw new GradleException(cleanupErrorMessage(), e);
                 }
             }
         }
+
+        private boolean suppressCleanupErrors() {
```

Comment:
I would have called these methods `isSuppressCleanupErrors()`, `getTestClass()`, etc., so the actual code could have been the same (i.e. could reference to the getters as if they were fields).

Reply:
Thanks, @wolfs. Interestingly enough I avoided the `get` prefix for that exact reason, that is, to keep the code somewhat similar to the original based on fields.

> i.e. could reference to the getters as if they were fields

This one is in Java.

Reply:
Ahhh, Java .... I knew I missed something.